samples/sensor/am2301b/Kconfig

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022 Hisanori YAMAOKU
+# SPDX-License-Identifier: Apache-2.0
+
+config LOG
+	default y
+
+config LOG_PRINTK
+	default y
+
+config SENSOR_LOG_LEVEL
+	default 4
+
+config I2C
+	default y
```

Comment:
please use `prj.conf` to enable necessary Kconfig options

Reply:
fixed it.